AY2122S1-CS2103T-T11-2/tp
# Fix validation regex

### Description

Fixed validation regex in `Phrase.java` to accept all foreign phrases and punctuation. This includes:
- Phrases with combining marks (e.g. umlautes, accents, etc.)
- Punctuation such as hyphens, fullstops, etc.

Added and edited test cases to ensure that these phrases and punctuation marks are accepted as valid.

Closes #76.

### Type of change


### How to test


- Run all tests
- Use the add and edit commands with any foreign phrase to make sure that the app accepts them

### Checklist
